### PR TITLE
Updates to PLAY-1141

### PR DESCRIPTION
**Pull request by thestephenmarshall**

### CODE REVIEW

File: playbook-website/app/javascript/components/Icons/index.tsx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from "react"
 
 import { Icon, Title, Flex, FlexItem, Card } from "playbook-ui"
-import Roofing from "!babel-loader!react-svg-loader!@powerhome/playbook-icons/icons/roofing.svg"
+import Roofing from "@powerhome/playbook-icons/icons/roofing.svg"
```
Comment (thestephenmarshall) [inline playbook-website/app/javascript/components/Icons/index.tsx:4]: Here, we are testing the import of an SVG React element in the view. It is working well.

File: playbook-website/config/webpack/environment.js [hunk 2]
```diff
@@ -13,6 +13,16 @@ environment.loaders.insert('javascript', {
...
 
+environment.loaders.insert('svgr', {
+  test: /\.(svg)$/,
+  use: {
+    loader: '@svgr/webpack',
+  },
+  include: [
+    path.resolve(__dirname, '../../../../node_modules/@powerhome/playbook-icons/icons')
+  ],
+})
```
Comment (thestephenmarshall) [inline playbook-website/config/webpack/environment.js:24]: Website only config that just says: "load SVGs from this directory as React elements"

File: playbook/app/pb_kits/playbook/pb_icon/_icon.tsx [hunk 3]
```diff
@@ -79,17 +79,16 @@ const Icon = (props: IconProps) => {
     [`fa-${size}`]: size,
     [`fa-pull-${pull}`]: pull,
     [`fa-rotate-${rotation}`]: rotation,
-
   }
 
-  const iconString = typeof icon === 'string'
-  const IconFunction = icon
-
+  const iconURL = typeof(icon) === 'string' && icon.includes('.svg') ? icon : null
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_icon/_icon.tsx:84]: We only care about the string format if it contains `.svg`.

File: playbook/app/pb_kits/playbook/pb_icon/_icon.tsx [hunk 4]
```diff
@@ -152,7 +151,19 @@ const Icon = (props: IconProps) => {
...
+          <span
+              {...dataProps}
+              {...htmlProps}
+              className={classesEmoji}
+              id={id}
+          >
+            <img src={iconURL} />
+          </span>
+        </>
+      )
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_icon/_icon.tsx:166]: When there is a url string for `icon` prop, use that on an image tag to render the SVG.

File: playbook/app/pb_kits/playbook/pb_icon/_icon.tsx [hunk 5]
```diff
@@ -135,11 +134,11 @@ const Icon = (props: IconProps) => {
               className="pbiconhere"
               id={id}
           >
-            <IconFunction />
+            { React.cloneElement(iconElement) }
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_icon/_icon.tsx:137]: Clone the element (allows for sending custom props like colors later) and render it as React element.

File: playbook/app/pb_kits/playbook/pb_icon/_icon.tsx [hunk 6]
```diff
@@ -79,17 +79,16 @@ const Icon = (props: IconProps) => {
...
-
+  const iconURL = typeof(icon) === 'string' && icon.includes('.svg') ? icon : null
   // Lets check and see if the icon prop is referring to a custom Power icon...
   // If so, then set fa-icon to "custom"
   // this ensures the JS will not do any further operations
   // faClasses[`fa-${icon}`] = customIcon ? 'custom' : icon
-  if (!customIcon && iconString) faClasses[`fa-${icon}`] = icon
+  if (!customIcon && !iconURL) faClasses[`fa-${icon}`] = icon as string
+
+  const iconElement: ReactSVGElement | null = typeof(icon) === "object" ? icon : null
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_icon/_icon.tsx:91]: Optionally, set this const if we got an true SVG React element.

File: playbook-website/config/webpack/environment.js (hunk 2 above)
Comment (markdoeswork) [reply playbook-website/config/webpack/environment.js:24]: 😍

**Pull request merged**